darts/models/forecasting/forecasting_model.py

```diff
@@ -628,6 +640,9 @@ def historical_forecasts(
             Note: Raises a ValueError if `start` yields a time outside the time index of `series`.
             Note: If `start` is outside the possible historical forecasting times, will ignore the parameter
             (default behavior with ``None``) and start at the first trainable/predictable point.
+        start_format
+            If set to 'index', `start` must be an integer and corresponds to the absolute position of the first point
```

Comment:
can we rephrase this similar to how pandas describes it with labels (e.g. values of index) for "point" and positions (e.g. to positional index) for "index".
Maybe we can even rename the accepted literals?

Reply:
Changed the name of the literal and tried to improve the phrasing, let me know what you think.